doc/manual/manual.bib

```diff
@@ -11315,7 +11315,7 @@ @misc{aspect-doi-v2.0.0
 }
 
 @article{aspectmanual,
-  title         = {{\textsc{ASPECT}: Advanced Solver for Problems in Earth's ConvecTion, User Manual}},
+  title         = {{\textsc{ASPECT}: {A}dvanced {S}olver for {P}roblems in {E}arth's {C}onvec{T}ion, User Manual}},
```

Comment:
this change is not necessary as there are already braces around the whole title.

Reply:
True, but there's also no harm here.